dlk/scripts/dlk_script_wrapper.sh

```diff
@@ -40,10 +40,16 @@ function make_each_target(){
 	if [[ "${1}" = "lm_x86" ]]
 	then
 	    strip "${2}"
+	elif [[ "${1}" = "lm_aarch64" ]]
+	then
+	        strip "${2}"  
```

Comment:
This line looks like `lm_x86` case.

 ```
elif [[ "${1}" = "lm_x86" ]] || [[ "${1}" = "lm_aarch64" ]]
    then
        strip "${2}"  
...
```
Is not enough?

Reply:
Indent level looks correct.

Reply:
Yes! You are right. In this case, we should use aarch64-linux-gnu-strip.

But I think this code is not good because of maintainability.
I recommend you to write "-Wl,-s" in Makefile (#104 )

These are equivalent, and latter is more simple.

Reply:
> I think this code is not good because of maintainability.

I see.